### PR TITLE
flashcards working

### PR DESCRIPTION
**Pull request by TomasKudlac**

### CODE REVIEW

File: app/views/card_sets/_question_form.html.erb [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
-  <%= f.submit "Submit", class: 'btn btn-primary' %>
-<% end %>
+<div class="flashcard">
+  <div>
+    <h2><%= flashcard.question %></h2>
+  </div>
+
+  <div class="flash-form">
+    <%= simple_form_for @answer, method: :patch, remote: true do |f| %>
+      <%= f.input :answer, as: :radio_buttons, collection: options %>
```
Comment (rahulkeerthi) [inline app/views/card_sets/_question_form.html.erb]: elegant, i like this!

File: app/views/card_sets/index.html.erb [hunk 2]
```diff
@@ -13,119 +13,56 @@
...
-              <%= render "set_card", { set: set, add_class: "set-completed"} %>
-            <% else %>
-              <!--logic for later, add something if user has attempted, but failed this set-->
-              <%= render "set_card", { set: set, add_class: "set-attempted"} %>
+              <% else %>
+                <%= render "set_card", set: set %>
+              <% end %>
+            </div>
+            <% if counter.even? %>
+              <%= render "set_card_info", set: @temp %>
```
Comment (rahulkeerthi) [inline app/views/card_sets/index.html.erb]: aw, breaks my css :(

File: app/views/card_sets/show.html.erb [hunk 3]
```diff
@@ -1,18 +1,29 @@
 <div>
-  <h1><%= @card_set.title %> - <%= @user_set.id %> - <%= @user_set.completed %></h1>
+  <h1><%= @card_set.title %></h1>
 </div>
 <div>
-  <% @user_set.user_answers.each do |answer| %>
-    <% @answers = [answer.flashcard.correct_answer, answer.flashcard.answer_1, answer.flashcard.answer_2, answer.flashcard.answer_3] %>
+  <% @card_set.flashcards.each do |flashcard| %>
```
Comment (rahulkeerthi) [inline app/views/card_sets/show.html.erb]: does this need to happen in the view?

Comment (TomasKudlac) [reply app/views/card_sets/show.html.erb]: needs to happen in the iteration and then passed on to form along with the coresponding flashcard

Comment (rahulkeerthi) [reply app/views/card_sets/show.html.erb]: 👍

File: app/views/card_sets/index.html.erb (hunk 2 above)
Comment (rahulkeerthi) [reply app/views/card_sets/index.html.erb]: after we merge this, i will go back to fix it properly

**Pull request merged**